### PR TITLE
Dict: More suggestions

### PR DESCRIPTION
**Pull request by sebweb3r**
inspired by @luzpaz

Looking for https://grep.app/search?q=posiblity revealed some more mistakes

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -24517,6 +24523,8 @@ serailization->serialization
 serailized->serialized
 serailze->serialize
 serailzed->serialized
+sercive->service
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:24526]: `sercived->serviced` ?

**Comment by lurch** [comment]
I'm not sure if it's worth adding similar misspellings for https://www.google.com/search?q=disservice ? There aren't any existing corrections for `disservice`, so maybe it's not a very commonly-used word. :shrug:

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24532,7 +24542,10 @@ sertificated->certificated
 sertificates->certificates
 sertification->certification
 servce->service
+servced->serviced
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:24545]: This should probably suggest `served` too? (with `servcing` also suggesting `serving`)

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -24532,7 +24542,10 @@ sertificated->certificated
...
 servces->services
+servcing->servicing
+servecing->servicing
 serveice->service
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:24549]: And perhaps this correction warrants the variants adding too? :)

Comment (lurch) [reply codespell_lib/data/dictionary.txt:24552]: Not sure why this was marked as resolved? I was suggesting adding `serveiced->serviced` etc.

**Comment by luzpaz** [comment]
LGTM

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:24552]: Because I misunderstood you :-)
I'll add your suggestions

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:24552]: 7882538c1af6b53a545f5fa5e1176a9cb7098bb0

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -20470,6 +20474,7 @@ posponed->postponed
 posption->position
 possabilites->possibilities
 possabilities->possibilities
+possability->possibility
 possabilties->possibilities
 possabily->possibly
 possable->possible
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:20480]: I wonder if `possable` should also suggest `posable` ? Might come up when e.g. writing about 3D animation software?
Although ironically, it seems a few occurrences of `posable` on grep.app are actually typos for `possible` :laughing:

Comment (lurch) [reply codespell_lib/data/dictionary.txt:20480]: (as always, feel free to ignore me if you think this is too unlikely)

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:20480]: i'm not sure about this one. But i found something, that I put in https://github.com/codespell-project/codespell/pull/1747

**Pull request merged**